src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.jsx

```diff
@@ -19,13 +19,14 @@ const RadioGroup = ({value, disabled, choices, label, onChange, ...props}) => {
           index={index}
           tabIndex={isSelected(id) ? 0 : -1}
           aria-checked={isSelected(id)}
+          disabled={disabled}
         >
           <RadioLineButton>
             {isSelected(id) && (
-              <RadioLineButtonFill isDisabled={disabled} animate={value !== ''} />
+              <RadioLineButtonFill disabled={disabled} animate={value !== ''} />
```

Comment:
I believe this will now produce DOM warnings, since emotion will pass isDisabled down to the DOM element, which divs cannot be disabled.

Reply:
Ok. I'll switch back to `isDisabled`. I had hoped to make the props consistent 😿

Reply:
There is a way to do it with `shouldForwardProp`, but it just ends up being more code defining the styled div.

Reply:
Wouldn't it be preferable to use `disabled` for consistency sake?

```javascript
const RadioLineButtonFill = styled(({disabled, animate, ...props}) => <div {...props} />)`
```